aws/sdk/integration-tests/s3/tests/query-strings-are-correctly-encoded.rs

```diff
@@ -60,7 +60,7 @@ async fn test_s3_signer_query_string_with_all_valid_chars() {
 
     // This is a snapshot test taken from a known working test result
     let snapshot_signature =
-        "Signature=775f88213304a5233ff295f869571554140e3db171a2d4a64f63902c49f79880";
+        "Signature=647aa91c7f91f1f1c498ef376fea370b48d0cd8c80a53c8e2cd64e3fc527a5e0";
```

Comment:
It must have been a pain to have to find all these tests. Thanks for doing all this.